src/features/foldingProvider.ts

```diff
@@ -99,16 +99,50 @@ export default class AsciidocFoldingRangeProvider implements vscode.FoldingRange
     }
   }
 
+  private static handleSingleLineCommentFoldingRanges (singleLineCommentStartIndexes: any[], foldingRanges: any[], lineIndex: number, lineText: string,
+    documentLineCount: number) {
+    if (lineText.slice(0, 2) === '//') {
```

Comment:
We can use `startsWith`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith